src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -420,3 +419,26 @@ def getPerformanceParameters(self, defaultParams):
         sizePerEvent = defaultParams.get('sizePerEvent', None) or 0
         memory = defaultParams.get('memoryRequirement', None) or 0
         return timePerEvent, sizePerEvent, memory
+    
+    def checkCreateFailConditoin(self, failedJob, failedReason):
+
+        valueMap = {"MaxWallTimeHours": "estimatedJobTime", 
+                    "MaxMemory": "estimatedMemoryUsage", 
```

Comment:
It's easier for us to read hours and GB, I agree! However, for eventual debugging and reading the code, it may be better to have the same unit used by condor. Anyways it's not a big deal, your call ;-)

About request_disk, I sent this question a long time ago to Tapas and I didn't get any answer back. To be honest, some magic happens and I don't know the trick :-D but I don't believe it's a bug since RequestDisk is nowhere in the code and the agent sets it properly.